readme.md

```diff
@@ -569,6 +569,7 @@
 - [Crypto Currency Tools & Algorithms](https://github.com/kennethreitz/awesome-coins) — Digital currency where encryption is used to regulate the generation of units and verify transfers.
 - [Open Source Supporters](https://github.com/zachflower/awesome-open-source-supporters) - Companies that offer their tools and services for free to open source projects.
 - [Design Principles](https://github.com/robinstickel/awesome-design-principles) - Create better and more consistent designs and experiences.
+- [Web Caching](https://github.com/zvaraondrej/awesome-web-cache) - Storing of a web resources.
```

Comment:
The title here is not consistent with the list name, and the description is not good enough. Caching is not just about storing a resource.